tests/docs/source/autoconfigurable/no_members.rst

```diff
@@ -0,0 +1,10 @@
+autoconfigurable - no members
+=============================
+
+Without the ``members`` option, one may expect no members to show up, but we
+present them all still, see `Issue27`_.
```

Comment:
I think it's a valid goal to present the option to export only the configurable options, and not the Python API of a class. I'm not saying `autoconfigurable` without members is necessarily the right signature, but that output should be an option.